src/YesSql.Core/Provider/IDbProviderOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using YesSql.Core.Services;
+
+namespace YesSql.Core.Provider
+{
+    public interface IDbProviderOptions
+    {
+        string ProviderName { get; set; }
```

Comment:
I got you now 😄, in that case we can add another extension that expose `UseXXX` without providing connection string and supply it from the settings inside the newly created method